Description: Fix -Werror=format-security compiler errors.
Author: Doug Torrance <dtorrance@monmouthcollege.edu>
Last-Update: 2014-10-18

--- a/src/wmcoincoin.c
+++ b/src/wmcoincoin.c
@@ -2174,7 +2174,7 @@
 
 
     if ((errmsg=dock_build_pixmap_porte(dock))) {
-      fprintf(stderr, errmsg);
+      fprintf(stderr, "%s", errmsg);
     }
 
 
--- a/src/prefs_gestion.c
+++ b/src/prefs_gestion.c
@@ -778,7 +778,7 @@
     /* MISE A JOUR DU COINCOIN */
     if (reset_dock_pix) { 
       if ((errmsg=dock_build_pixmap_porte(dock))) {
-	fprintf(stderr, errmsg);
+	fprintf(stderr, "%s", errmsg);
       }
       close_palmi = 1;
     }
