Description: Fix -Wtype-limits compiler warning.
Author: Doug Torrance <dtorrrance@monmouthcollege.edu>
Last-Update: 2014-10-24

--- a/src/pinnipede.c
+++ b/src/pinnipede.c
@@ -599,7 +599,11 @@
   attr = 0;
 
   if (pv->is_plopified == 3) p = "plop"; /* bienvenue dans le monde de la hard plopification */
+#ifdef NO_BITFIELDS
   assert(pv->is_plopified <= 3);
+#else
+  ;
+#endif
 
   has_initial_space = 1;
   while (p) {
