Description: Fix -Wunused-variable compiler warnings.
Author: Doug Torrance
Last-Update: 2014-10-19

--- a/src/fontcoincoin.c
+++ b/src/fontcoincoin.c
@@ -244,7 +244,6 @@
   XGlyphInfo ext;
   if (len == -1) len = strlen(str);
   XftTextExtents8(display, ccfonts[fid]->xfn, str, len, &ext);
-  int plop = ext.width;
   XftTextExtentsUtf8(display, ccfonts[fid]->xfn, str, len, &ext);
   return ext.width;
 }
--- a/src/kbcoincoin.c
+++ b/src/kbcoincoin.c
@@ -54,7 +54,7 @@
       rlen = Xutf8LookupString(ic, event, buf, buf_len, &kb_state()->ksym, &status);
     }
   }
-  unsigned i;
+  //unsigned i;
   switch (status) {
     case XLookupNone:
       //printf("XLookupNone\n");
