Description: Fix -Wunused-parameter compiler warnings.
Author: Doug Torrance <dtorrance@monmouthcollege.edu>
Last-Update: 2014-10-24

--- a/src/wmccc_callbacks.c
+++ b/src/wmccc_callbacks.c
@@ -13,7 +13,8 @@
 on_bottom1_activate                    (GtkMenuItem     *menuitem,
                                         gpointer         user_data)
 {
-
+  (void) menuitem;
+  (void) user_data;
 }
 
 
@@ -21,6 +22,7 @@
 on_left1_activate                      (GtkMenuItem     *menuitem,
                                         gpointer         user_data)
 {
-
+  (void) menuitem;
+  (void) user_data;
 }
 
--- a/src/wmccc_support.c
+++ b/src/wmccc_support.c
@@ -80,6 +80,8 @@
   gchar *pathname = NULL;
   GtkWidget *pixmap;
 
+  (void) widget;
+
   if (!filename || !filename[0])
       return gtk_image_new ();
 
